src/plugins/graph/components/editable-graph-value.tsx

```diff
@@ -0,0 +1,125 @@
+import { observer } from 'mobx-react';
+import React, { useState, useRef, useEffect } from 'react';
+import { AxisBounds, AxisPlace } from '../imports/components/axis/axis-types';
+import { useAxisLayoutContext } from '../imports/components/axis/models/axis-layout-context';
+
+import "./editable-graph-value.scss";
+
+interface IEditableValueProps {
+  value: number;
+  minOrMax: "min" | "max";
+  axis: AxisPlace;
+  onValueChange: (newValue: number) => void;
+  readOnly?: boolean;
+}
+
+export const EditableGraphValue: React.FC<IEditableValueProps> = observer(function EditableGraphValue(props) {
+  const { value, minOrMax, axis, onValueChange, readOnly } = props;
+  const [isEditing, setIsEditing] = useState(false);
+  const inputRef = useRef<HTMLInputElement | null>(null);
+  const borderBoxRef = useRef<HTMLInputElement | null>(null);
+  const layout = useAxisLayoutContext();
+  const axisBounds = layout.getComputedBounds(axis) as AxisBounds;
+
+  useEffect(() => {
+    if (isEditing && inputRef.current) {
+      inputRef.current.select();
+    }
+  }, [isEditing]);
+
+  //**************  Dynamically calculate border box size and offset positions ******************
+  useEffect(() => {
+    if (borderBoxRef.current) {
+      // Calculate the width of the border box (and Y axis left offset) depending on how many characters are in value
+      const numOfCharacters = value.toString().length;
+      const widthPerCharacter = 8;
+      let boxWidth: number;
+      if (numOfCharacters === 1){
+        boxWidth = 15;
+      } else if (numOfCharacters === 2){
+        boxWidth = 20;
+      } else {
+        boxWidth = value.toString().length * widthPerCharacter;
+      }
+      borderBoxRef.current.style.width = `${boxWidth}px`;
+
+      // For left axis determine min/max left offset based on axisBounds and width of border box
+      let leftOffset;
+      if (axis === 'left') {
+        const yTickRightEdgePosition = axisBounds.width - 6; //represents right edge of each Y tick
```

Comment:
Based on line 69 of `use-sub-axis.ts`, I feel like this should be `axisBounds.left + axisBounds.width - 6`. Is there a reason you left out the `axisBounds.left`?

Reply:
True, this is 0 in all cases I just tested now - but I'll include it as our conversation yesterday.